src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -39,3 +39,10 @@ export function makeHumanReadable(inputStr) {
     .map(word => word[0].toUpperCase() + word.substr(1).toLowerCase())
     .join(' ');
 }
+
+// Helper to detect if we're on review page when we don't have access
+// to form context. Necessary because list and loop pages don't seem
+// to respect 'hideOnReview' or 'keepInPageOnReview'
```

Comment:
If this is something you think might be a bug with the forms library, it might be worth filing an issue for it. https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new/choose